ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_statistical.py

```diff
@@ -486,3 +486,44 @@ def test_jax_numpy_max(
         keepdims=keepdims,
         where=where,
     )
+
+
+# average
+@handle_frontend_test(
+    fn_tree="jax.numpy.average",
+    dtype_x_axis=statistical_dtype_values(function="average"),
+    returned=st.booleans(),
+)
+def test_jax_numpy_average(
+    *,
+    dtype_x_axis,
+    returned,
+    num_positional_args,
+    with_out,
+    as_variable,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    x_dtype, x, axis = dtype_x_axis
+    
+    if isinstance(axis, tuple):
+        axis = axis[0]
```

Comment:
Does this not fiter out axis data like (1,2). I believe we should test for this as jax.numpy's average function does take tuple of ints as inputs also.

Reply:
Thank you for reviewing.  I am not able to pass the test for tuple cases. The message ver long and here is the last part of the message
```
def _average(a, axis: Optional[Union[int, Tuple[int, ...]]] = None, weights=None,
                 returned=False):
      if weights is None: # Treat all weights as 1
        _check_arraylike("average", a)
        a, = _promote_dtypes_inexact(a)
        avg = mean(a, axis=axis)
        if axis is None: 
          weights_sum = lax.full((), core.dimension_as_value(a.size), dtype=avg.dtype)
        else:
>         weights_sum = lax.full_like(avg, core.dimension_as_value(a.shape[axis]), dtype=avg.dtype)
E         jax._src.traceback_util.UnfilteredStackTrace: TypeError: tuple indices must be integers or slices, not tuple
E         
E         The stack trace below excludes JAX-internal frames.
E         The preceding is the original exception that occurred, unmodified.
```
In the latest document of Jax, there is an extra `elif` branch, it is to extract the shape of the array by arg `axis` as value and calculate the size by prod

```
def _average(a: ArrayLike, axis: Axis = None, weights: Optional[ArrayLike] = None,
             returned: bool = False, keepdims: bool = False) -> Union[Array, Tuple[Array, Array]]:
  if weights is None: # Treat all weights as 1
    _check_arraylike("average", a)
    a, = _promote_dtypes_inexact(a)
    avg = mean(a, axis=axis, keepdims=keepdims)
    if axis is None:
      weights_sum = lax.full((), core.dimension_as_value(a.size), dtype=avg.dtype)
    elif isinstance(axis, tuple):
      weights_sum = lax.full_like(avg, _prod(core.dimension_as_value(a.shape[d]) for d in axis))
    else:
      weights_sum = lax.full_like(avg, core.dimension_as_value(a.shape[axis]))  # type: ignore[index]
```